test/metabase/test/generate.clj

```diff
@@ -238,6 +238,14 @@
                                               :database_id   [:database :id]
                                               :table_id      [:table :id]
                                               :collection_id [:collection :id]}}
+   ;; like card but is a model and the query is very simple
```

Comment:
Would be good to document why we need this here as well

Reply:
added a line under this : https://github.com/metabase/metabase/pull/29021/commits/1d8c6582972870b47b21603782d8618639ef371d